src/components/checkTodo.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="status-container">
+    <img
+      class="unchecked"
+      v-if="!checked"
+      @click="toggleCheck"
+      src="../assets/unchecked.png"
+    />
+    <div class="checked-todo" v-else @click="toggleCheck">
+      <img class="circle" src="../assets/checked.png" />
+      <img class="vector" src="../assets/Vector.png" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref } from "vue";
+
+const checked = ref(false);
+
+function toggleCheck() {
+  checked.value = !checked.value;
+  console.log(checked.value);
```

Comment:
Always make sure that you remove all the unnecessary `console.log` before you create the PR